<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputDialog</class>
 <widget class="QDialog" name="OutputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>478</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2650</width>
    <height>1240</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Employee Attendance System</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/icon.png</normaloff>:/icon/icon.png</iconset>
  </property>
  <widget class="QLabel" name="img_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>371</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>411</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="clock_in_btn">
      <property name="text">
       <string>Clock In</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clock_out_btn">
      <property name="text">
       <string>Clock Out</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>221</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Time :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="time_label">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Date :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="date_label">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>221</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Details</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>161</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Status :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="status_label">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Name :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="name_label">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
